examples/vnet/doca_dpdk.c

```diff
@@ -793,6 +812,9 @@ static int doca_dpdk_build_fwd(struct doca_dpdk_pipe *pipe,
 	case DOCA_FWD_RSS:
 		doca_dpdk_build_rss_action(action_entry, fwd_cfg);
 		break;
+    case DOCA_FWD_PORT:
```

Comment:
here you need to consult with some local cfg object. if doca_dpdk_cfg.nic? then hairpin, else port

Reply:
this is done in doca_dpdk_build_fwd_action. we pass it a bool parameter if true then hairpin.
i added that bool in the fwd_cfg struct that is set upon the init, but can be changed i think this should be in the application domain.
what do you think ?